### DISCUSSION TITLE
Multi-Lingual Support and Localization

### DISCUSSION THREAD

**Discussion by rosie-at-pieces** [pieces-app/support#141] [Ideas]
Today, Pieces only supports English in its user interface. Let us know what languages you would like to see us support!

**Comment by eumel1968** [comment]
German would be a nice to have.

**Comment by HippieStereo** [comment]
FR - French 
PT - Portuguese of Portugal 

I speak both if one day you want help with that I can help 👍

**Comment by dividedby-0** [comment]
I would like to suggest adding Italian as a language option, and I am willing to contribute to the translation efforts.

**Comment by yegor-pelykh** [comment]
I believe having **Russian** language available in the interface would be very useful. I'm prepared to assist with these changes.

**Comment by vibestudio-git** [comment]
I hope Pieces will provide a language option as soon as possible, and I wish Korean language support to be added. If there is anything I can do to help with this, I am happy to support it.

**Comment by faithleysath** [comment]
as chinese, i would like to see **Simplified Chinese**. Willing to provide any assistance.

**Comment by HazrinaAdelia** [comment]
LT- Lithuania, please!!!!!!

**Comment by norifumi3** [comment]
I really hope that Japanese will be supported.

**Comment by norifumi3** [comment]
Following up on my previous comment about Japanese support - I'd like to share a specific use case that makes language support particularly important.

  The "Single-Click Summary" feature in Pieces Desktop is excellent, but it currently generates summaries only in English. For non-English speakers like myself, this
  significantly reduces the usefulness of the feature.

  **Feature Request:**
  - Add a language setting option for AI-generated content (summaries, descriptions, etc.)
  - Allow users to choose their preferred output language

  This would make Pieces much more accessible to the global developer community. I believe many users from the comments above (Korean, Chinese, Russian, etc.) would
  also benefit from this enhancement.

  Thank you for considering this!

**Comment by belikovi861-oss** [comment]
Russian, please !/ Вообще для такого инструмента очень нужен выбор языка.
